src/components/cards/CreatePool/PreviewPool.vue

```diff
@@ -1,21 +1,38 @@
 <script lang="ts" setup>
```

Comment:
Can we rename this PreviewPoolModal.vue, the word modal should be used in modal component names.